vividus-plugin-azure-service-bus/src/main/java/org/vividus/azure/servicebus/service/ServiceBusService.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.azure.servicebus.service;
+
+import com.azure.core.amqp.AmqpRetryOptions;
+import com.azure.core.credential.TokenCredential;
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusException;
+import com.azure.messaging.servicebus.ServiceBusMessage;
+import com.azure.messaging.servicebus.ServiceBusSenderClient;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.vividus.azure.servicebus.ServiceBusSendMessageException;
+import org.vividus.azure.servicebus.model.ServiceBusType;
+
+public class ServiceBusService
+{
+    private static final Logger LOGGER = LoggerFactory.getLogger(ServiceBusService.class);
+
+    private ServiceBusClientBuilder clientBuilder;
+
+    public ServiceBusService(TokenCredential tokenCredential, AmqpRetryOptions amqpRetryOptions)
+    {
+        this.clientBuilder = new ServiceBusClientBuilder()
+                .credential(tokenCredential)
+                .retryOptions(amqpRetryOptions);
+    }
+
+    public void send(ServiceBusType type, String name, String namespaceName, String message)
+    {
+        ServiceBusClientBuilder.ServiceBusSenderClientBuilder senderBuilder = clientBuilder
```

Comment:
is `clientBuilder` resuable? in other words would it allow to create a multiple `senderBuilder`-s?

Reply:
yes, it is